modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * monitoring/enabling-monitoring-for-user-defined-projects.adoc
 
+:_mod-docs-content-type: CONCEPT
```

Comment:
Good catch!

Reply:
This was actually @Srivaralakshmi's catch! :slightly_smiling_face:  She is really good at spotting these :+1: